features/git-town/configuration/configuration_automatic.feature

```diff
@@ -27,13 +27,31 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     | git sync-fork      |
 
 
+  Scenario Outline: Proceeding to configuration upon initial config prompt (using prompt's default)
+    When I run `<COMMAND>` and enter "" and "^C"
+    Then I see "Git Town hasn't been configured for this repository."
```

Comment:
Yes. I think we already have a spec that verifies that the config wizard comes up if the configuration is missing. At least we should have one. This particular spec only describes the default value for the first prompt. So it can assume that the config wizard works without testing it again, and just verifies that hitting Enter on the first prompt does the right thing.